src/Interpreters/executeQuery.cpp

```diff
@@ -163,6 +163,10 @@ static void logQuery(const String & query, const Context & context, bool interna
             (current_user != "default" ? ", user: " + current_user : ""),
             (!initial_query_id.empty() && current_query_id != initial_query_id ? ", initial_query_id: " + initial_query_id : std::string()),
             (context.getSettingsRef().use_antlr_parser ? "new" : "old"),
+            (!context.getSettingsRef().log_comment.empty()
```

Comment:
But format string is not changed.

Reply:
Should we concatenate the comment and the query? For example, if the `query` is `select * from t;`, and if `log_comment` is set, the query should be `comment: select * from t`?

Reply:
No, query and comment should be unrelated.
Just add the corresponding placeholder `{}` to the format string.

Reply:
So should it be `select * from t comment: ...`?

Reply:
So should it be `SELECT * FROM Customers -- WHERE City='Berlin';`